pkg/command/plugin.go

```diff
@@ -39,7 +39,7 @@ var (
 )
 
 const (
-	invalidTargetMsg                = "invalid target specified. Please specify a correct value for the `--target/-t` flag from '" + common.TargetList + "'"
+	invalidTargetMsg                = "invalid target specified. Please specify a correct value for the `--target` flag from '" + common.TargetList + "'"
```

Comment:
`-t` is valid for the `--target` flag for the `plugin` sub-cmds.  But I'm find not putting `-t` in the error message.  I'm just mentioning it because you removed it at the same time as you removed it for the `--target` flag for contexts, but the two cases are different.

Reply:
For the error message, you don't have to put the `-t`